src/js/components/organisms/sub-nav/sub-nav.html

```diff
@@ -1,16 +1,15 @@
-<nav aria-label="sub items" class="sub-nav sub-nav--fixed" :class="classObject" v-if="page">
-    <faux-scroll>
+<nav aria-label="sub items" class="sub-nav sub-nav--fixed" ref="subNav" v-if="page">
+    <faux-vertical-scroll>
         <ul class="sub-nav__list" role="presentation"  v-if="page.links && page.links.siblings && page.links.siblings.length">
-            <li class="sub-nav__item" v-for="sibling in page.links.siblings">
-                <router-link class="sub-nav__link" :to="mapLink(sibling.href)" active-class="sub-nav__link--active">
-                    <responsive-picture class="sub-nav__image" v-if="sibling.icon"
+            <li class="sub-nav__item" v-for="sibling in page.links.siblings" ref="listItems">
```

Comment:
Can you please add the :key on the v-for?
Or instead of "key", you can use it like this: 

v-for= "(sibling, index) in page.links.siblings"

Reply:
It only needs a a key when you do a v-for on vue components (not on html elements)